packages/config/src/tokens/generated.json

```diff
@@ -7257,14 +7257,14 @@
       "chainId": 34443,
       "source": "external",
       "supply": "totalSupply",
+      "excludeFromTotal": true,
       "bridgedUsing": {
         "bridges": [
           {
             "name": "XLink bridge"
           }
         ]
-      },
-      "excludeFromTotal": true
+      }
     },
```

Comment:
what happened here in generated.json with the other tokens?

Reply:
wow weird